models/cow_protocol/ethereum/cow_protocol_ethereum_order_rewards.sql

```diff
@@ -1,16 +1,18 @@
 {{ config(alias='order_rewards',
```

Comment:
can you make the alias property:
`alias=alias('order_rewards'),`

notes from readme:
```
update alias property again, this time to simplify and remove legacy flag
alias = alias('blocks'),
```

this leverages a new alias macro that helps us differentiate table naming in our metastores

Reply:
this will occur in all new models for dunesql, so the three total steps:
- tag for dunesql
- alias macro in config block
- translated syntax

Reply:
So, always use alias macro. No problemo - just need to go change all my PRs.

Reply:
Done here.

Reply:
Update -- also now done in all open PRs.

Reply:
we assign a default value for the legacy flag in the macro to `False`, so that's why we don't use on these translated ones, and only override in legacy files

Reply:
Sounds good to me... as long as everything is correct now.